test/e2e/framework/psp/psp.go

```diff
@@ -149,8 +152,16 @@ func createPrivilegedPSPBinding(f *Framework, namespace string) {
 				Name:      "default",
 			})
 		ExpectNoError(err)
-		ExpectNoError(auth.WaitForNamedAuthorizationUpdate(f.ClientSet.AuthorizationV1(),
+		ExpectNoError(auth.WaitForNamedAuthorizationUpdate(kubeClient.AuthorizationV1(),
 			serviceaccount.MakeUsername(namespace, "default"), namespace, "use", podSecurityPolicyPrivileged,
 			schema.GroupResource{Group: "extensions", Resource: "podsecuritypolicies"}, true))
 	}
 }
+
+// ExpectNoError is a copy from the same name function in file test/e2e/framework.go
+func ExpectNoError(err error, explain ...interface{}) {
+	if err != nil {
+		e2elog.Logf("Unexpected error occurred: %v", err)
+	}
+	gomega.ExpectWithOffset(2, err).NotTo(gomega.HaveOccurred(), explain...)
```

Comment:
the `ExpectNoError` from the framework calls `gomega.ExpectWithOffset(1, err)` not `gomega.ExpectWithOffset(2, err)`, correct? If so, we should change this.
This patch changes the current behavior. Which is not wrong but we should add it in a separate PR and only focus on refactoring for this on. This way we can revert if something starts failing in CI.

/lgtm cancel

Reply:
@alejandrox1  Do you mean we should change  `gomega.ExpectWithOffset(2, err).NotTo(gomega.HaveOccurred(), explain...)` to` gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred(), explain...)`? But I believe the original definition should be  `gomega.ExpectWithOffset(2, err).NotTo(gomega.HaveOccurred(), explain...)`

Reply:
Sorry, you are completely right!
Given https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L1321
a `framework.NoExpectError` is equivalent to `gomega.ExpectWithOffset(2, err).NotTo(gomega.HaveOccurred(), explain...)`.

/lgtm
/hold cancel